pkg/mounter/utils/helper.go

```diff
@@ -31,6 +31,15 @@ func computeMountPathHash(target string) string {
 	return rand.SafeEncodeString(fmt.Sprint(hasher.Sum32()))
 }
 
+func computeVolumeIdLabelVal(volumeId string) string {
+	errs := validation.IsValidLabelValue(volumeId)
+	if errs == nil {
+		return volumeId
+	}
+	volSha := sha256.Sum256([]byte(volumeId))
+	return hex.EncodeToString(volSha[:])[:63]
```

Comment:
Since we are not caring about collision attack. Can we just use sha1, which is 40 hex chars, to avoid the somewhat confusing `[:63]`? And I also advise adding a `h1.` prefix, to clearly indicate it is a hash.
```suggestion
	volSha := sha1.Sum([]byte(volumeId))
	return "h1." + hex.EncodeToString(volSha[:])
```